recipes/r-mmcpcounter/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: r-mmcpcounter
+  version: 1.1.0
```

Comment:
It is good practice to store the version in a variable, and then reference the variable both here and in `url`. This avoids making mistakes when the package is updated. 

See, for instance, https://github.com/bioconda/bioconda-recipes/blob/076c5aa81e36b8bf875ebff07f330b2b04f40ce5/recipes/r-epic/meta.yaml#L1-L10